plugins/jenkins-backend/src/index.ts

```diff
@@ -21,3 +21,4 @@
  */
 
 export * from './service';
+export { jenkinsPlugin } from './plugin';
```

Comment:
After [this PR ](https://github.com/backstage/backstage/pull/19822) I would vouche for a quick fix here. You couldn't have known as it probably happended in parallel :)

```suggestion
export { jenkinsPlugin as default } from './plugin';
```

Reply:
updated